prebid_cache_client/client.go

```diff
@@ -181,6 +185,26 @@ func encodeValueToBuffer(value Cacheable, leadingComma bool, buffer *bytes.Buffe
 		buffer.WriteString(string(value.Key))
 		buffer.WriteString(`"`)
 	}
+
+	//vtrack specific
+	if len(value.BidID) > 0 {
```

Comment:
Please add test coverage for the cache client's new `BidID`,` Bidder`, and `Timestamp` members.